pkg/apis/kops/instancegroup.go

```diff
@@ -103,6 +105,15 @@ type InstanceGroupSpec struct {
 	Taints []string `json:"taints,omitempty"`
 }
 
+type MachineSpec struct {
```

Comment:
The struct doesn't much matter, it is the field name (and strictly only the json form of it)